react/features/base/tracks/functions.js

```diff
@@ -35,7 +35,7 @@ export async function createLocalPresenterTrack(options, desktopHeight) {
         video: {
             aspectRatio: 4 / 3,
             height: {
-                exact: result
+                ideal: result
```

Comment:
out of curiosity, why this change?

Reply:
Sorry, should have mentioned the reason for the change. As per spec, when we specify exact, the property must have to be be considered to find a match while with ideal, it is considered optional. I am trying to avoid situations where GUM would fail because the camera doesn't support that particular resolution.

Reply:
Makes sense, thanks!